src/test/java/pages/BasePage.java

```diff
@@ -15,31 +17,46 @@ public class BasePage {
     public WebDriver driver;
     public WebDriverWait wait;
     public Actions actions;
```

Comment:
I recommend: make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

For example

```
    protected WebDriver driver;
    protected WebDriverWait wait;
    protected Actions actions;
```